tests/test_quickbooks_automatic_fields.py

```diff
@@ -114,4 +110,9 @@ def test_run(self):
                 # Verify the number or records exceeds the max_results (api limit)
                 pagination_threshold = int(self.get_properties().get(page_size_key))
                 self.assertGreater(record_count, pagination_threshold,
-                                   msg="Record count not large enough to gaurantee pagination.")
+                                   msg="Record count not large enough to guarantee pagination.")
```

Comment:
This assertion deviates from the test objectives and should be covered in the pagination test, please remove it.

Reply:
done